server/odc-test/src/main/java/com/oceanbase/odc/test/database/TestDBType.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oceanbase.odc.test.database;
+
+import lombok.NonNull;
+
+/**
+ * @author gaoda.xy
+ * @date 2023/9/27 13:53
+ */
+public enum TestDBType {
```

Comment:
prefer not public scope

Reply:
This enum may be used by unit test in other module, so i make it public